code/modules/mob/living/living.dm

```diff
@@ -1186,14 +1186,13 @@
 	var/has_legs = get_num_legs()
 	var/has_arms = get_num_arms()
 	var/paralyzed = IsParalyzed()
-	var/knockdown = IsKnockdown()
```

Comment:
You should make the IsKnockdown function have SHOULD_NOT_OVERRIDE(TRUE)

Reply:
added in aac50c8